library/globals.inc.php

```diff
@@ -610,6 +611,12 @@
        '0',                             // default
       xl('Restrict non-authorized users to the Schedule Facilities set in User admin.')
     ),
+        'facility_acl' => array(
+      xl('Restrict User access by Facility'),
+       'bool',                          // data type
+       '0',                             // default
+      xl('Restrict User access to patients by assigned patient facility.')
+    ),
```

Comment:
When the role based menus are utilized this should be a function of a setting in the user field. Just a note.